src/Service/Sns/tests/Unit/Input/CreatePlatformEndpointInputTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace AsyncAws\Sns\Tests\Unit\Input;
+
+use AsyncAws\Core\Test\TestCase;
+use AsyncAws\Sns\Input\CreatePlatformEndpointInput;
+
+class CreatePlatformEndpointInputTest extends TestCase
+{
+    public function testRequest(): void
+    {
+        self::fail('Not implemented');
+
+        $input = new CreatePlatformEndpointInput([
+            'PlatformApplicationArn' => 'change me',
+            'Token' => 'change me',
+            'CustomUserData' => 'change me',
+            'Attributes' => ['change me' => 'change me'],
+        ]);
+
+        // see https://docs.aws.amazon.com/sns/latest/APIReference/API_CreatePlatformEndpoint.html
```

Comment:
right link is https://docs.aws.amazon.com/sns/latest/api/API_CreatePlatformEndpoint.html

Reply:
and you can use the snippet provided in the doc:
```
https://sns.us-west-2.amazonaws.com/?Action=CreatePlatformEndpoint
&PlatformApplicationArn=arn%3Aaws%3Asns%3Aus-west-2%3A123456789012%3Aapp%2FGCM%2Fgcmpushapp
&Token=APA91bGi7fFachkC1xjlqT66VYEucGHochmf1VQAr9k...jsM0PKPxKhddCzx6paEsyay9Zn3D4wNUJb8m6HZrBEXAMPLE
&CustomUserData=UserId%3D27576823
&Version=2010-03-31
&AUTHPARAMS 
```

Reply:
Maybe it could. But I suggest doing that in a second PR.

Reply:
Sadly AWS is inconsistent in naming. And, as far as I know, the link to the Documentation cannot be guessed.
hegenerated link works for several services, but not all. :(

Reply:
Could they be added to the manifest.json?

Reply:
yes I think that would be the best solution.

Reply:
I added an issue #687